earth_enterprise/src/common/packet.h

```diff
@@ -812,15 +814,15 @@ struct etPolyLinePacket : public etDataPacket
 
 #ifdef FORCE_BASE32
 struct etAreaPacketData_BASE32 {
-  uint32           name;
+  std::uint32_t           name;
   int          texId;
   ushort       numPt;
   ushort       bitFlags;
 
 #ifdef VECTOR_COMPRESSION
-  uint32 localPt;
+  std::uint32_t localPt;
 #else
-  uint32 localPt;
+  std::uint32_t localPt;
```

Comment:
last comment here, it looks like in just about every situation where the `FORCE_BASE32` and `VECTOR_COMPRESSION` macros are defined, `localPt` will always be defined as a 32 bit integer and the guards can be removed. unrelated to this pr, but i feel this file should be looked at, at some point down the road